### PR TITLE
logging: shorten precommit log message

### PR DESCRIPTION
**Pull request by tychoish**
This is an attempt to clean up the logging message as requested in #6269.

### CODE REVIEW

**Review by ValarDragon — APPROVED** [review]
This LGTM! Our logs shall now be much smaller =)

File: types/vote_set.go [hunk 1]
```diff
@@ -560,6 +562,22 @@ func (voteSet *VoteSet) StringShort() string {
...
+	if voteSet == nil {
+		return nilVoteSetString
+	}
+	voteSet.mtx.Lock()
+	defer voteSet.mtx.Unlock()
+	voted, total, frac := voteSet.sumTotalFrac()
+
+	return fmt.Sprintf("Votes:%d/%d(%.3f) Validators:%q %v",
+		voted, total, frac,
+		voteSet.valSet,
```
Comment (marbar3778) [inline types/vote_set.go:577]: This might produce a large log when the validator set is large.

File: types/vote_set.go [hunk 2]
```diff
@@ -560,6 +562,22 @@ func (voteSet *VoteSet) StringShort() string {
...
+		voteSet.votesBitArray)
```
Comment (marbar3778) [inline types/vote_set.go:578]: not sure if this adds value in logs, does it? cc @alexanderbez

File: types/vote_set.go (hunk 1 above)
Comment (tychoish) [reply types/vote_set.go:577]: True! I thought about that, but this has always been in the message content so we're not making it longer, and the prototype in the original data included some of this data as well, so I presumed that it was useful. I'm down to take it out, particularly if @ValarDragon thinks that would be useful.

We could also only put it in if the total number of votes was small or if the fraction was small, if that would be useful.

File: types/vote_set.go (hunk 2 above)
Comment (tychoish) [reply types/vote_set.go:578]: Definitely! It was in the prototype "new" long message in the original ticket, but I'm down to remove it.

File: types/vote_set.go [hunk 3]
```diff
@@ -560,6 +562,22 @@ func (voteSet *VoteSet) StringShort() string {
...
+// vote set.
+func (voteSet *VoteSet) LogString() string {
+	if voteSet == nil {
+		return nilVoteSetString
+	}
+	voteSet.mtx.Lock()
+	defer voteSet.mtx.Unlock()
+	voted, total, frac := voteSet.sumTotalFrac()
+
+	return fmt.Sprintf("Votes:%d/%d(%.3f) Validators:%q %v",
```
Comment (ValarDragon) [inline types/vote_set.go:575]: I wonder if only the cumulative fraction is useful to have here

File: types/vote_set.go (hunk 2 above)
Comment (ValarDragon) [reply types/vote_set.go:578]: I don't think it is super valuable. It is pretty fun to look at when watching logs, but not sure if thats a good reason to keep it

File: types/vote_set.go (hunk 1 above)
Comment (ValarDragon) [reply types/vote_set.go:577]: I'm not really clear on what this is

File: types/vote_set.go (hunk 3 above)
Comment (tychoish) [reply types/vote_set.go]: I just updated it to the fraction + float without the other info

File: types/vote_set.go (hunk 1 above)
Comment (tychoish) [reply types/vote_set.go]: Same, same. Easy to get rid of it!

File: types/vote_set.go (hunk 2 above)
Comment (tychoish) [reply types/vote_set.go]: great, I took it out :)

**Review by marbar3778 — APPROVED** [review]
utACK. Love cleaner logs.

**Review by alexanderbez — APPROVED** [review]

**Comment by alexanderbez** [comment]
Excellent @tychoish! I think we should back-port this to 0.34.x. What do you think @marbar3778?

**Pull request merged**